README.md

```diff
@@ -6,63 +6,158 @@
+chi2_stat = pqm_chi2(p, q, re_tessellation = 1000)
+print(np.mean(chi2_stat), np.std(chi2_stat)) # 98.51, 11.334
 
 # To get pvalues from PQMass
-pvalues = pqm_pvalue(x_sample, y_sample, num_refs = 100, re_tessellation = 50)
-print(np.mean(pvalues), np.std(pvalues))
+pvalues = pqm_pvalue(p, q, re_tessellation = 1000)
+print(np.mean(pvalues), np.std(pvalues)) # 0.50, 0.26
+```
+
+We see that both $\chi_{PQM}^2$ and $\text{p-value}(\chi_{PQM}^2)$ follow the expected $\chi^2$ indicatiing that both $x$ and $y$ come from the same underlying distribution.
+
+Another such example in which we do $\textit{not}$ expect $x$ and $y$ to come from the same distribution is if $x$ is again sampled from a $\mathcal{N}(0, 1)$ in 10 dimensions whereas $y$ is sampled from a $\mathcal{U}(0, 1)$ in 10 dimensions.
+
+```python
+from pqm import pqm_pvalue, pqm_chi2
+import numpy as np
+
+p = np.random.normal(size = (500, 10))
+q = np.random.uniform(size = (400, 10))
 
 # To get chi^2 from PQMass
-chi2_stat = pqm_chi2(x_sample, y_sample, num_refs = 100, re_tessellation = 50)
-print(np.mean(chi2_stat), np.std(chi2_stat))
+chi2_stat = pqm_chi2(p, q, re_tessellation = 1000)
+print(np.mean(chi2_stat), np.std(chi2_stat)) # 577.29, 25.74
+
+# To get pvalues from PQMass
+pvalues = pqm_pvalue(p, q, re_tessellation = 1000)
+print(np.mean(pvalues), np.std(pvalues)) # 3.53e-56, 8.436e-55
 ```
 
-If your two samples are drawn from the same distribution, then the p-value
-should be drawn from the random uniform(0,1) distribution. This means that if
-you get a very small value (i.e., 1e-6), then you have failed the null
-hypothesis test, and the two samples are not drawn from the same distribution.
-If you get values approximately equal to 1 every time then that suggests
-potential duplication of samples between `x_samples` and `y_samples`.
+Here it is clear that both $\chi_{PQM}^2$ and $\text{p-value}(\chi_{PQM}^2)$ are not close to their expected results, thus showing that $x$ and $y$ do $\textbf{not}$ come from the same underlying distribution.
 
-For the chi^2 metric, given your two sets of samples, if they come from the same
-distribution, the histogram of your chi^2 values should follow the chi^2
-distribution. The degrees of freedom (DoF) will equal `DoF = num_refs - 1` The
-peak of this distribution will be at `DoF - 2`, the mean will equal `DoF`, and
-the standard deviation will be `sqrt(2 * DoF)`. If your chi^2 values are too
-high (`chi^2 / DoF > 1`), it suggests that the samples are out of distribution.
-Conversely, if the values are too low (`chi^2 / DoF < 1`), it indicates
-potential duplication of samples between `x_samples` and `y_samples` (i.e.
-memorization for generative models).
+Thus, PQMass can be used to identify if any two distributions come from the same underlying distributions if enough samples are given. We encourage users to look through the paper to see the varying experiments and use cases for PQMass!
+
+## How to Intrept Result
```

Comment:
Maybe this could be part of an FAQ? It's not really about interpreting the results generally, but more about a specific case that can go wrong